wordle-web/src/components/LetterButton.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <LetterBase :char="props.letter.char" :color="letter.color"></LetterBase>
+</template>
+
+<script setup lang="ts">
```

Comment:
We've just been adding to the same branch since the start of the class, we haven't pulled the changes from the main repo which is probably what has been causing these problems.